chain/src/store.rs

```diff
@@ -378,6 +378,13 @@ impl<'a> Batch<'a> {
 			db: self.db.child()?,
 		})
 	}
+
+	/// An iterator to all lived block now
+	pub fn iter_lived_blocks(&self) -> Result<SerIterator<Block>, Error> {
```

Comment:
Maybe rename this to `blocks_iter()` for consistency with `difficulty_iter()` (and others)?

Reply:
Ok.